TransferMethod/Tests/ListTransferMethodPresenterTests.swift

```diff
@@ -9,6 +9,8 @@ import XCTest
 class ListTransferMethodPresenterTests: XCTestCase {
     private var presenter: ListTransferMethodPresenter!
     private let mockView = MockListTransferMethodView()
```

Comment:
rename MockListTransferMethodView to ListTransferMethodViewMock

Reply:
fixed